### PR TITLE
update terminology based on recent IBM Style Guide word list

### PR DESCRIPTION
**Pull request by samccann**
##### SUMMARY

IBM Style guide recommends the changes in this PR for word use.
NOTE - this is not a comprehensive list of changes based on the style guide, this is only the newer term recommendations from 2020 through Q2 2021.

##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME

docs/docsite/rst/dev_guide/developing_collections_contributing.rst
##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: docs/docsite/rst/network/user_guide/network_best_practices_2.5.rst [hunk 1]
```diff
@@ -318,23 +318,23 @@ You can also look at the backup files:
 If `ansible-playbook` fails, please follow the debug steps in :ref:`network_debug_troubleshooting`.
 
 
-.. _network-agnostic-examples:
+.. _network-independent-examples:
 
-Example 2: simplifying playbooks with network agnostic modules
-==============================================================
+Example 2: simplifying playbooks with network independent modules
```
Comment (acozine) [inline docs/docsite/rst/network/user_guide/network_best_practices_2.5.rst]: This doesn't quite have the same meaning in my ear, plus I could see it being hard to translate. I'm not sure what to suggest, though. Maybe "multi-network" or "multi-vendor"?

**Review by acozine — COMMENTED** [review]
Overall this is great. Using `use` instead of `leverage` is much simpler and clearer. I like getting rid of `agnostic` also, but I think for some of these instances we may need to think about rephrasing rather than substituting `independent` in the existing phrases.

File: docs/docsite/rst/network/user_guide/network_best_practices_2.5.rst (hunk 1 above)
Comment (samccann) [reply docs/docsite/rst/network/user_guide/network_best_practices_2.5.rst]: Reading the content, I think platform independent works well. The point is that it is a generic 'cli_command' module that can work with multiple vendor products.

Comment (samccann) [inline docs/docsite/rst/network/user_guide/network_best_practices_2.5.rst]:
```suggestion
Example 2: simplifying playbooks with platform independent modules
```

File: docs/docsite/rst/network/user_guide/network_best_practices_2.5.rst [hunk 2]
```diff
@@ -318,23 +318,23 @@ You can also look at the backup files:
...
 
 (This example originally appeared in the `Deep Dive on cli_command for Network Automation <https://www.ansible.com/blog/deep-dive-on-cli-command-for-network-automation>`_ blog post by Sean Cavanaugh -`@IPvSean <https://github.com/IPvSean>`_).
 
-If you have two or more network platforms in your environment, you can use the network agnostic modules to simplify your playbooks. You can use network agnostic modules such as ``ansible.netcommon.cli_command`` or ``ansible.netcommon.cli_config`` in place of the platform-specific modules such as ``arista.eos.eos_config``, ``cisco.ios.ios_config``, and ``junipernetworks.junos.junos_config``. This reduces the number of tasks and conditionals you need in your playbooks.
+If you have two or more network platforms in your environment, you can use the network independent modules to simplify your playbooks. You can use network independent modules such as ``ansible.netcommon.cli_command`` or ``ansible.netcommon.cli_config`` in place of the platform-specific modules such as ``arista.eos.eos_config``, ``cisco.ios.ios_config``, and ``junipernetworks.junos.junos_config``. This reduces the number of tasks and conditionals you need in your playbooks.
```
Comment (samccann) [inline docs/docsite/rst/network/user_guide/network_best_practices_2.5.rst]:
```suggestion
If you have two or more network platforms in your environment, you can use the platform independent modules to simplify your playbooks. You can use platform independent modules such as ``ansible.netcommon.cli_command`` or ``ansible.netcommon.cli_config`` in place of the platform-specific modules such as ``arista.eos.eos_config``, ``cisco.ios.ios_config``, and ``junipernetworks.junos.junos_config``. This reduces the number of tasks and conditionals you need in your playbooks.
```

File: docs/docsite/rst/network/user_guide/network_best_practices_2.5.rst [hunk 3]
```diff
@@ -318,23 +318,23 @@ You can also look at the backup files:
...
 
 .. note::
-  Network agnostic modules require the :ref:`ansible.netcommon.network_cli <ansible_collections.ansible.netcommon.network_cli_connection>` connection plugin.
+  Network independent modules require the :ref:`ansible.netcommon.network_cli <ansible_collections.ansible.netcommon.network_cli_connection>` connection plugin.
```
Comment (samccann) [inline docs/docsite/rst/network/user_guide/network_best_practices_2.5.rst]:
```suggestion
  Platform independent modules require the :ref:`ansible.netcommon.network_cli <ansible_collections.ansible.netcommon.network_cli_connection>` connection plugin.
```

File: docs/docsite/rst/network/user_guide/network_best_practices_2.5.rst [hunk 4]
```diff
@@ -318,23 +318,23 @@ You can also look at the backup files:
...
 
 
 Sample playbook with platform-specific modules
 ----------------------------------------------
 
-This example assumes three platforms, Arista EOS, Cisco NXOS, and Juniper JunOS.  Without the network agnostic modules, a sample playbook might contain the following three tasks with platform-specific commands:
+This example assumes three platforms, Arista EOS, Cisco NXOS, and Juniper JunOS.  Without the network independent modules, a sample playbook might contain the following three tasks with platform-specific commands:
```
Comment (samccann) [inline docs/docsite/rst/network/user_guide/network_best_practices_2.5.rst]:
```suggestion
This example assumes three platforms, Arista EOS, Cisco NXOS, and Juniper JunOS.  Without the platform independent modules, a sample playbook might contain the following three tasks with platform-specific commands:
```

File: docs/docsite/rst/network/user_guide/network_best_practices_2.5.rst [hunk 5]
```diff
@@ -354,10 +354,10 @@ This example assumes three platforms, Arista EOS, Cisco NXOS, and Juniper JunOS.
       commands: show interface
     when: ansible_network_os == 'vyos.vyos.vyos'
 
-Simplified playbook with ``cli_command`` network agnostic module
-----------------------------------------------------------------
+Simplified playbook with ``cli_command`` network independent module
```
Comment (samccann) [inline docs/docsite/rst/network/user_guide/network_best_practices_2.5.rst]:
```suggestion
Simplified playbook with ``cli_command`` platform independent module
```

File: docs/docsite/rst/network/user_guide/network_best_practices_2.5.rst [hunk 6]
```diff
@@ -354,10 +354,10 @@ This example assumes three platforms, Arista EOS, Cisco NXOS, and Juniper JunOS.
...
+-------------------------------------------------------------------
 
-You can replace these platform-specific modules with the network agnostic ``ansible.netcommon.cli_command`` module as follows:
+You can replace these platform-specific modules with the network independent ``ansible.netcommon.cli_command`` module as follows:
```
Comment (samccann) [inline docs/docsite/rst/network/user_guide/network_best_practices_2.5.rst]:
```suggestion
You can replace these platform-specific modules with the platform independent ``ansible.netcommon.cli_command`` module as follows:
```

File: docs/docsite/rst/network/user_guide/network_best_practices_2.5.rst [hunk 7]
```diff
@@ -420,7 +420,7 @@ If you use groups and group_vars by platform type, this playbook can be further
         register: command_output
 
 
-You can see a full example of this using group_vars and also a configuration backup example at `Network agnostic examples <https://github.com/network-automation/agnostic_example>`_.
+You can see a full example of this using group_vars and also a configuration backup example at `Network independent examples <https://github.com/network-automation/agnostic_example>`_.
```
Comment (samccann) [inline docs/docsite/rst/network/user_guide/network_best_practices_2.5.rst]:
```suggestion
You can see a full example of this using group_vars and also a configuration backup example at `Platform independent examples <https://github.com/network-automation/agnostic_example>`_.
```

File: docs/docsite/rst/network/user_guide/network_best_practices_2.5.rst [hunk 8]
```diff
@@ -318,23 +318,23 @@ You can also look at the backup files:
 If `ansible-playbook` fails, please follow the debug steps in :ref:`network_debug_troubleshooting`.
 
 
-.. _network-agnostic-examples:
+.. _network-independent-examples:
 
-Example 2: simplifying playbooks with network agnostic modules
-==============================================================
+Example 2: simplifying playbooks with platform independent modules
```
Comment (samccann) [inline docs/docsite/rst/network/user_guide/network_best_practices_2.5.rst]:
```suggestion
Example 2: simplifying playbooks with platform-independent modules
```

File: docs/docsite/rst/network/user_guide/network_best_practices_2.5.rst [hunk 9]
```diff
@@ -318,23 +318,23 @@ You can also look at the backup files:
...
 (This example originally appeared in the `Deep Dive on cli_command for Network Automation <https://www.ansible.com/blog/deep-dive-on-cli-command-for-network-automation>`_ blog post by Sean Cavanaugh -`@IPvSean <https://github.com/IPvSean>`_).
 
-If you have two or more network platforms in your environment, you can use the network agnostic modules to simplify your playbooks. You can use network agnostic modules such as ``ansible.netcommon.cli_command`` or ``ansible.netcommon.cli_config`` in place of the platform-specific modules such as ``arista.eos.eos_config``, ``cisco.ios.ios_config``, and ``junipernetworks.junos.junos_config``. This reduces the number of tasks and conditionals you need in your playbooks.
+If you have two or more network platforms in your environment, you can use the platform independent modules to simplify your playbooks. You can use platform independent modules such as ``ansible.netcommon.cli_command`` or ``ansible.netcommon.cli_config`` in place of the platform-specific modules such as ``arista.eos.eos_config``, ``cisco.ios.ios_config``, and ``junipernetworks.junos.junos_config``. This reduces the number of tasks and conditionals you need in your playbooks.
```
Comment (samccann) [inline docs/docsite/rst/network/user_guide/network_best_practices_2.5.rst]:
```suggestion
If you have two or more network platforms in your environment, you can use the platform-independent modules to simplify your playbooks. You can use platform-independent modules such as ``ansible.netcommon.cli_command`` or ``ansible.netcommon.cli_config`` in place of the platform-specific modules such as ``arista.eos.eos_config``, ``cisco.ios.ios_config``, and ``junipernetworks.junos.junos_config``. This reduces the number of tasks and conditionals you need in your playbooks.
```

File: docs/docsite/rst/network/user_guide/network_best_practices_2.5.rst [hunk 10]
```diff
@@ -318,23 +318,23 @@ You can also look at the backup files:
...
 
 .. note::
-  Network agnostic modules require the :ref:`ansible.netcommon.network_cli <ansible_collections.ansible.netcommon.network_cli_connection>` connection plugin.
+  Platform independent modules require the :ref:`ansible.netcommon.network_cli <ansible_collections.ansible.netcommon.network_cli_connection>` connection plugin.
```
Comment (samccann) [inline docs/docsite/rst/network/user_guide/network_best_practices_2.5.rst]:
```suggestion
  Platform-independent modules require the :ref:`ansible.netcommon.network_cli <ansible_collections.ansible.netcommon.network_cli_connection>` connection plugin.
```

File: docs/docsite/rst/network/user_guide/network_best_practices_2.5.rst [hunk 11]
```diff
@@ -318,23 +318,23 @@ You can also look at the backup files:
...
 
 
 Sample playbook with platform-specific modules
 ----------------------------------------------
 
-This example assumes three platforms, Arista EOS, Cisco NXOS, and Juniper JunOS.  Without the network agnostic modules, a sample playbook might contain the following three tasks with platform-specific commands:
+This example assumes three platforms, Arista EOS, Cisco NXOS, and Juniper JunOS.  Without the platform independent modules, a sample playbook might contain the following three tasks with platform-specific commands:
```
Comment (samccann) [inline docs/docsite/rst/network/user_guide/network_best_practices_2.5.rst]:
```suggestion
This example assumes three platforms, Arista EOS, Cisco NXOS, and Juniper JunOS.  Without the platform-independent modules, a sample playbook might contain the following three tasks with platform-specific commands:
```

File: docs/docsite/rst/network/user_guide/network_best_practices_2.5.rst [hunk 12]
```diff
@@ -354,10 +354,10 @@ This example assumes three platforms, Arista EOS, Cisco NXOS, and Juniper JunOS.
       commands: show interface
     when: ansible_network_os == 'vyos.vyos.vyos'
 
-Simplified playbook with ``cli_command`` network agnostic module
-----------------------------------------------------------------
+Simplified playbook with ``cli_command`` platform independent module
```
Comment (samccann) [inline docs/docsite/rst/network/user_guide/network_best_practices_2.5.rst]:
```suggestion
Simplified playbook with ``cli_command`` platform-independent module
```

File: docs/docsite/rst/network/user_guide/network_best_practices_2.5.rst [hunk 13]
```diff
@@ -354,10 +354,10 @@ This example assumes three platforms, Arista EOS, Cisco NXOS, and Juniper JunOS.
...
+-------------------------------------------------------------------
 
-You can replace these platform-specific modules with the network agnostic ``ansible.netcommon.cli_command`` module as follows:
+You can replace these platform-specific modules with the platform independent ``ansible.netcommon.cli_command`` module as follows:
```
Comment (samccann) [inline docs/docsite/rst/network/user_guide/network_best_practices_2.5.rst]:
```suggestion
You can replace these platform-specific modules with the platform-independent ``ansible.netcommon.cli_command`` module as follows:
```

File: docs/docsite/rst/network/user_guide/network_best_practices_2.5.rst [hunk 14]
```diff
@@ -420,7 +420,7 @@ If you use groups and group_vars by platform type, this playbook can be further
         register: command_output
 
 
-You can see a full example of this using group_vars and also a configuration backup example at `Network agnostic examples <https://github.com/network-automation/agnostic_example>`_.
+You can see a full example of this using group_vars and also a configuration backup example at `Platform independent examples <https://github.com/network-automation/agnostic_example>`_.
```
Comment (samccann) [inline docs/docsite/rst/network/user_guide/network_best_practices_2.5.rst]:
```suggestion
You can see a full example of this using group_vars and also a configuration backup example at `Platform-independent examples <https://github.com/network-automation/agnostic_example>`_.
```

File: docs/docsite/rst/network/user_guide/network_best_practices_2.5.rst (hunk 1 above)
Comment (acozine) [reply docs/docsite/rst/network/user_guide/network_best_practices_2.5.rst]: +1 and thanks for adding the hyphen, I think that will help with translation

File: docs/docsite/rst/network/user_guide/network_best_practices_2.5.rst [hunk 15]
```diff
@@ -318,23 +318,23 @@ You can also look at the backup files:
 If `ansible-playbook` fails, please follow the debug steps in :ref:`network_debug_troubleshooting`.
 
 
-.. _network-agnostic-examples:
+.. _network-independent-examples:
 
-Example 2: simplifying playbooks with network agnostic modules
-==============================================================
+Example 2: simplifying playbooks with platform-independent modules
+=================================================================
```
Comment (acozine) [inline docs/docsite/rst/network/user_guide/network_best_practices_2.5.rst]: one character too short:
```suggestion
==================================================================
```

File: docs/docsite/rst/network/user_guide/network_best_practices_2.5.rst [hunk 16]
```diff
@@ -354,10 +354,10 @@ This example assumes three platforms, Arista EOS, Cisco NXOS, and Juniper JunOS.
       commands: show interface
     when: ansible_network_os == 'vyos.vyos.vyos'
 
-Simplified playbook with ``cli_command`` network agnostic module
-----------------------------------------------------------------
+Simplified playbook with ``cli_command`` platform-independent module
+-------------------------------------------------------------------
```
Comment (acozine) [inline docs/docsite/rst/network/user_guide/network_best_practices_2.5.rst]: here too:
```suggestion
--------------------------------------------------------------------
```

**Pull request merged**